mln/services/friend.py

```diff
@@ -70,6 +70,9 @@ def handle_friend_invite_response(user, relation_id, accept):
 def remove_friend(user, relation_id):
 	"""
 	Remove a friend from the user's friend list.
+	Raise RuntimeError if
+	- the friendship does not exist
+	- the relation does not relate to this user, or

```

Comment:
Just one tiny thing - with the deletion of the following line, the ,` or` should be moved up one line.